src/System.Private.CoreLib/src/System/Reflection/MdImport.cs

```diff
@@ -397,17 +397,16 @@ public unsafe string GetUserString(int mdToken)
             if (name == null)
                 return null;
 
+#if ALIGN_ACCESS
```

Comment:
@jkotas, I couldn't find anywhere that this is set.  Is this dead code?

Reply:
It used to be set set when we were running on platforms with very strict alignment requirements (Itanium, PPC, Sparc, ...). We not support such platforms today, and even if we did support them - the code would still work given how the string constructor is implemented (it does not actually require strict alignment).

I think you can delete it.

Reply:
Ok, thanks.